pkg/storage/flamegraph_test.go

```diff
@@ -151,46 +198,81 @@ func TestGenerateFlamegraph(t *testing.T) {
 		Children: []*pb.FlamegraphNode{{
 			Meta: &pb.FlamegraphNodeMeta{
 				Function: &pb.Function{
+					Id:   "00000000-0000-0000-0000-000000000000",
 					Name: "1",
 				},
-				Line:     &pb.Line{},
-				Location: &pb.Location{},
+				Line: &pb.Line{
+					LocationId: "00000000-0000-0000-0000-000000000000",
+					FunctionId: "00000000-0000-0000-0000-000000000000",
+				},
+				Location: &pb.Location{
+					Id:        "00000000-0000-0000-0000-000000000000",
+					MappingId: "00000000-0000-0000-0000-000000000000",
+				},
```

Comment:
That actually happens because I was just too lazy to update the inputs (so they defaulted to empty UUIDs. I agree though that this looks quite confusing and I had to go back and understand what's happening as well, so I'll update the inputs appropriately.

Reply:
Elaborating more on line & location being nil. Lines is already a slice and it will only be filled if location is known and there is actually a function, so checking for nil there doesn't make sense. However, we do already have the case where there are no lines, and in that case we don't fill function and line at all.